.vscode/settings.json

```diff
@@ -20,5 +23,7 @@
       "url": "./.vscode/package.json-schema.json"
     }
   ],
+  "tailwindCSS.classFunctions": ["classnames", "classNames"],
```

Comment:
Enables Tailwind IntelliSense to work with both casing variants (classnames/classNames) of the classnames library